apps/module-scan/src/store.ts

```diff
@@ -333,6 +344,7 @@ export default class Store {
     }
 
     await this.dbCreate()
+    this.batchNumber = 0
```

Comment:
This happens to have the same effect because of the `!this.batchNumber` truthy check, but I think it'd be better to do `delete this.batchNumber`. That said, I think we'll probably be better off with an auto-incrementing number for this.